definitions.yaml

```diff
@@ -76,6 +76,23 @@ definitions:
                       default:
                         description: Default size of the docker volume, if not specified
                         type: number
+              kubelet:
+                description: Information about the kubelet volume attached to the worker node. This volume is used to store files for the kubelet (volume mounts etc)
```

Comment:
Please append a period to `etc` and also to the end of the sentence.

Reply:
How about mentioning that this is also used by `EmptyDir` type volumes?

Reply:
added. thanks!